data/transaction.go

```diff
@@ -23,17 +23,19 @@ type SendTransactionsResponse struct {
 
 // Transaction holds the fields of a transaction to be broadcasted to the network
 type Transaction struct {
-	Nonce     uint64 `json:"nonce"`
-	Value     string `json:"value"`
-	RcvAddr   string `json:"receiver"`
-	SndAddr   string `json:"sender"`
-	GasPrice  uint64 `json:"gasPrice,omitempty"`
-	GasLimit  uint64 `json:"gasLimit,omitempty"`
-	Data      []byte `json:"data,omitempty"`
-	Signature string `json:"signature,omitempty"`
-	ChainID   string `json:"chainID"`
-	Version   uint32 `json:"version"`
-	Options   uint32 `json:"options,omitempty"`
+	Nonce             uint64 `json:"nonce"`
+	Value             string `json:"value"`
+	RcvAddr           string `json:"receiver"`
+	SndAddr           string `json:"sender"`
+	GasPrice          uint64 `json:"gasPrice,omitempty"`
+	GasLimit          uint64 `json:"gasLimit,omitempty"`
+	Data              []byte `json:"data,omitempty"`
+	Signature         string `json:"signature,omitempty"`
+	ChainID           string `json:"chainID"`
+	Version           uint32 `json:"version"`
+	Options           uint32 `json:"options,omitempty"`
```

Comment:
```suggestion
	Options           uint32 `json:"options"`
```

Reply:
We still need omitempty for backwards compatibility

Reply:
:+1: